pkgs/racket-doc/scribblings/reference/syntax.scrbl

```diff
@@ -2094,11 +2094,14 @@ followed by a parallel sequence of booleans to indicate whether each
 optional-argument value was provided.}
 
 
-@defform/subs[(case-lambda [formals body ...+] ...)
+@deftogether[(
+@defform[(case-lambda [formals body ...+] ...)]
+@defform/subs[(case-λ [formals body ...+] ...)
               ([formals (id ...)
                         (id ...+ . rest-id)
-                        rest-id])]{
-               
+                        rest-id])]
+)]{
+
 Produces a procedure. Each @racket[[formals body ...+]]
 clause is analogous to a single @racket[lambda] procedure; applying
 the @racket[case-lambda]-generated procedure is the same as applying a
```

Comment:
Please add `@history[#:changed "8.13.0.5" @elem{Added @racket[case-λ].}]` before the closing curly brace for this form's documentation (and I will bump the Racket version number soon afterward).